src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs

```diff
@@ -38,16 +47,6 @@ protected HttpResponse Response
             get { return Context.Response; }
         }
 
-        protected PathString OriginalPathBase { get; private set; }
-
-        protected PathString OriginalPath { get; private set; }
-
-        protected ILogger Logger { get; private set; }
-
-        protected UrlEncoder UrlEncoder { get; private set; }
-
-        public IAuthenticationHandler PriorHandler { get; set; }
```

Comment:
I don't think its worth preserving things in the Handler just for obsolete, since they will see the error already from all of the UseXyzs